### PR TITLE
Feat/home

### PR DESCRIPTION
**Pull request by biswanath-tewari**
features:
1. navbar
2. home screen

### CODE REVIEW

File: src/components/Navbar/index.jsx [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+        <NavLink
+          to={'/videos'}
+          text="explore"
+          toggleNav={toggleNav}
+          isOpen={isOpen}
+          isBtn={false}
+          icon="fa-film"
+        />
+
+        <NavLink
```
Comment (sharathnair9999) [inline src/components/Navbar/index.jsx:111]: Liked the Custom NavLink you've created

**Review by sharathnair9999 — APPROVED** [review]
As usual the code looks pretty clean. Good work

**Review by SukanyaSen530 — APPROVED** [review]
Liked the custom Navlink component.

File: src/components/Navbar/styles.scss [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+.navbar--transparent {
+  background-color: transparent !important;
+
+  .e-logo {
+    max-width: 7rem;
+    max-height: 7rem;
+    cursor: pointer;
+  }
+
+  .e-logo-small {
```
Comment (SukanyaSen530) [inline src/components/Navbar/styles.scss]: Since this is a variation of logo, you can keep the name as e-logo--small and nest it inside e-logo.

Comment (biswanath-tewari) [reply src/components/Navbar/styles.scss]: Thanks for the feedback, missed it.

**Pull request merged**